README.md

```
@@ -77,15 +77,28 @@ To ensure our algorithm captures true cultural relevance:
-Create a `.env` file in the project root:
+1. Copy the example environment file:
 
 ```bash
-# Required API keys
-TMDB_API_KEY=your_tmdb_api_key_here
-OMDB_API_KEY=your_omdb_api_key_here
+cp .env.example .env
+```
+
+2. Edit `.env` and add your API keys:
+   - Get TMDB API key from https://www.themoviedb.org/settings/api
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)